rake/build/package_st2.rake

```diff
@@ -33,6 +33,7 @@ namespace :package do
             opts.components.each do |component|
               execute :cp, "packages/st2/component.makefile ${GITDIR}/#{component}/Makefile"
             end
+	    execute :bash, '$GITDIR/scripts/populate-sha.sh'
```

Comment:
Looks like scripts like that logically belong to current `st2-packages` repo, rather then `st2` repo, since it's part of packaging. IMO.